projects/schematics/src/ng-add/actions/configs.ts

```diff
@@ -24,7 +24,7 @@ const IFRAME_SILENT_RENEW = `{
 
 const AZURE_AD_SILENT_RENEW = `{
             authority: 'https://login.microsoftonline.com/<authorityUrlOrTenantId>/v2.0',
-            authWellknownEndpoint: 'https://login.microsoftonline.com/common/v2.0',
+            authWellknownEndpointUrl: 'https://login.microsoftonline.com/common/v2.0',
```

Comment:
While testing the migration this gave a compile error, so I renamed this property to match the type.